src/pool/srv_target.c

```diff
@@ -737,6 +737,36 @@ ds_pool_start(uuid_t uuid)
 	return rc;
 }
 
+/*
+ * Called via dss_thread_collective() to stop all container services
```

Comment:
[Nit] Container Service is cont_svc, which is very different from ds_cont_child. I think it's clear to just say "ds_cont_child".